app/components/modal-confirm-email-send.js

```diff
@@ -10,6 +12,19 @@ export default ModalComponent.extend({
     // Allowed actions
     confirm: () => {},
 
+    deliveredToMessage: computed('model.{paidOnly,memberCount}', function () {
+        const isEditor = this.get('session.user.isEditor');
+        if (this.model.paidOnly) {
```

Comment:
@gargol Thinking its probably a good idea to add an extra layer of check here for `this.model` being defined ? 🤔

Reply:
@gargol Updated